.eslintrc

```diff
@@ -0,0 +1,60 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "prettier",
+    "universe"
+  ],
+  "parser": "@babel/eslint-parser",
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "parserOptions": {
+    "requireConfigFile": false,
+    "babelOptions": {
+      "presets": ["@babel/preset-react"]
+    },
+    "ecmaFeatures": {
+        "jsx": true
+    },
+    "ecmaVersion": 12,
+    "sourceType": "module",
+    "parserOptions": {
+      "project": "./tsconfig.json"
+    }
+  },
+  "plugins": [
+    "react",
+    "jest",
+    "testing-library",
+    "no-only-tests",
+    "sort-keys-fix",
```

Comment:
i'm all for alphabetizing , i'm curious will this sort keys alphabetically for us automatically or just throw an error..?